libs/utils/src/services/auth/getAuth.ts

```diff
@@ -0,0 +1,43 @@
+import { API_URL } from '@wsvvrijheid/config'
+import { SessionUser } from '@wsvvrijheid/types'
+import { Auth } from '@wsvvrijheid/types'
+import axios from 'axios'
+
+export const getAuth: (
+  identifier: string,
+  password: string,
+) => Promise<Auth | undefined> = async (
+  identifier: string,
+  password: string,
+) => {
+  const response = await axios.post(`${API_URL}/api/auth/local`, {
```

Comment:
We can maybe create a AuthResponse type for login response in `libs/types/src/auth.ts`

```ts
type AuthResponse = {
  jwt: string;
  user: {
    id: number;
    username: string;
    email: string;
    provider: string;
    confirmed: boolean;
    blocked: boolean;
    createdAt: string;
    updatedAt: string;
  };
};
```

```suggestion
  const response = await axios.post<AuthResponse>(`${API_URL}/...
```

Reply:
There are also artist, avatar, volunteer props in the response. Instead of adding this should I use existing User type and add non-existing props?
New suggestion: 
type AuthResponse = {
  jwt: string
  user?: User | null
  id: number | null
  createdAt: string | null
  updatedAt: string | null
}

Reply:
I'm not sure if the `post` request returns extended user data. We might need to adjust the backend for `/api/auth/local` request. Populated user data returns for the request `/api/users`